api/swagger-spec/oapi-v1.json

```diff
@@ -29196,6 +29196,10 @@
      "wildcardPolicy": {
       "type": "string",
       "description": "Wildcard policy is the wildcard policy that was allowed where this route is exposed."
+     },
+     "routerCanonicalHostname": {
+      "type": "string",
+      "description": "CanonicalHostname is the external host name for the router that can be used as a CNAME for the host requested for this route. This value is optional and may not be set in all cases."
```

Comment:
CanonicalHostname is the external host name for the router that can be used as a CNAME for the host requested for this route.
=>
CanonicalHostname is the external host name of the router.  It can be used by the person creating the route as the target of a CNAME if the route contains a host for the DNS that they control.

Perhaps?